uk.co.compucorp.civicrm.hrleaveandabsences/CRM/HRLeaveAndAbsences/Page/LeaveRequestCalendarFeedConfig.php

```diff
@@ -166,4 +170,93 @@ private function calculateLinksMask($leaveCalendarFeedConfig) {
 
     return $mask;
   }
+
+  /**
+   * Returns the text to be displayed for the composed_of or visible_to
+   * filter fields in the calendar feed listing table.
+   *
+   * @param array $filterParams
+   *
+   * @return string
+   */
+  private function getFilterDisplayText($filterParams) {
+    $displayText = '';
+    foreach($filterParams as $field => $value) {
+      if ($field == 'department') {
+        $departmentNames = $this->getOptionValuesList('hrjc_department', $value);
+        $displayText .= 'Department: '. implode(', ' , $departmentNames) . '. ';
+      }
+
+      if ($field == 'location') {
+        $locationNames = $this->getOptionValuesList('hrjc_location', $value);
+        if ($displayText) {
+          $displayText .= '<br/>';
+        }
+        $displayText .= 'Location: '. implode(', ' , $locationNames) . '.';
```

Comment:
Kindly add  a space around the concatenation
`$displayText .= 'Location: ' . implode(', ' , $locationNames) . '.';`